Update spacing between user dropdown items

arathjz opened the pull request:
**What:**
Update spacing between user dropdown items

**Why:**
Closes: https://app.asana.com/0/1159164196409156/1195530778765025/f

**How:**
- Reducing the size of dropdown items
- Add margin-bottom to the logout button

### Media
 ![Image](http://g.recordit.co/TGBrsqLkUA.gif)

duranmla commented:
Hey BTW, I think we should change the color of the icon on hover too, isn't?
_It is commented on the ticket too._

duranmla reviewed (changes requested):
Also it seems that the design suggests a border of the avatar when the dropdown is active. Should we include it?

arathjz commented:
@duranmla I addressed your comments, this is how the dropdown looks like now. I will go ahead and close this PR in order to update the header in the apps that uses it

![image](https://user-images.githubusercontent.com/20747535/98753786-c60afd00-238a-11eb-85eb-85a0dd289e2f.png)
